hue/hue.sh

```diff
@@ -20,34 +20,36 @@ ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
 
 # Only run on the master node of the cluster
 if [[ "${ROLE}" == 'Master' ]]; then
```

Comment:
It's not usual in our init actions but exiting 0 if we are not on the master saves a lot of indenting.

Reply:
done